backend/audit/views.py

```diff
@@ -33,3 +41,15 @@ def fetch_my_subnissions(cls, user):
             .filter(submitted_by=user)
         )
         return data
+
+
+class EditSubmission(LoginRequiredMixin, generic.View):
+    redirect_field_name = "Home"
+
+    def get(self, request, *args, **kwargs):
+
+        # template_name = "audit/edit_submission.html"
+        report_id = kwargs["report_id"]
+        # sac = SingleAuditChecklist.objects.get(report_id=report_id)
+        # return render(request, template_name, context)
```

Comment:
[NB] Should these commented out lines still be there? I'm wondering if they were tested code (so should be removed) or code that might have been commented out for testing (in which maybe they should be uncommented).

Reply:
That's fine. It might be helpful to leave in an extra line saying that too.